api/v2/checluster_types.go

```diff
@@ -228,10 +228,10 @@ type DevfileRegistry struct {
 // Configuration settings related to the database used by the Che installation.
 // +k8s:openapi-gen=true
 type Database struct {
-	// Instructs the Operator to deploy a dedicated database.
+	// Instructs the operator to deploy a dedicated database.
 	// By default, a dedicated PostgreSQL database is deployed as part of the Che installation.
 	// When `externalDb` is set as `true`, no dedicated database is deployed by the
-	// Operator and you need to provide connection details to the external DB you want to use.
+	// operator and you need to provide connection details to the external database you want to use.
```

Comment:
```suggestion
	// Operator. You must provide the connection details of the external database you want to use.
```
- Consider stating this user action as a separate sentence. 
- It's unclear whether the action is to provide the information _to_ or _about_ the database. I think you mean about.